docs/pages/database-access/auto-user-provisioning/database-access-controls.mdx

```diff
@@ -0,0 +1,213 @@
+---
+title: Database Access Controls
+description: Understand the concepts behind Database Access Controls managed by Teleport.
+---
+
+**Database Access Controls** is a Teleport feature that lets you configure
+role-based access controls for database objects. A **database
+object** can be a table, view, or stored procedure. With Database Access Controls, you can ensure that users only have permissions to manage the data
+they need.
+
+You can define **import rules** that instruct the Teleport Database Service to
+apply labels to database objects when it encounters these objects in a user
+session. When a user connects to a database, the Database Service allows or
+denies the session by checking database object labels against the user's
+Teleport roles.
```

Comment:
I don't think this paragraph is true. The database objects are imported independent of what the user does within the session. The session is always allowed but only granted privileges to those database objects matched against the user's Teleport roles.

Reply:
Thanks, you are right this was somewhat inaccurate. PTAL: https://github.com/gravitational/teleport/pull/39109/commits/b1fff4893d44be8c14b7c12bb2a354d0796f3cd0

Reply:
Uh, you are right, I must have dropped my editor hat today...